todo_app/app.py

```diff
@@ -1,19 +1,35 @@
 from flask import Flask
 from flask import render_template, request, redirect
 
+from dotenv import load_dotenv
+
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items, add_item
+from todo_app.data.trello_items import get_items, add_item, change_ticket_list
+
+load_dotenv()
```

Comment:
You should find this line is completely unnecessary - since we have the `python-dotenv` library installed Flask actually executes this automatically when we run `poetry run flask run`
(And there may be times we _don't_ want it to run automatically, such as when we're setting up the environment differently for automated tests)

Reply:
Good to know, thanks